rasgotransforms/rasgotransforms/transforms/outliers/outliers.yaml

```diff
@@ -0,0 +1,56 @@
+name: outliers
+tags:
+  - row
+  - math
+  - conditional
+description: |
+  This function determines which records in the table are an outlier based on a given 
+  statistical method (z-score, IQR, or manual threshold) and a target column. It produces 
+  a new column named 'OUTLER_<target_column>' which is TRUE for records that are outliers, 
+  and FALSE for records that aren't.
+arguments:
+  target_columns:
+    type: column_list
+    description: The target columns containing values which will be used to calculate outliers. 
+      The z-score or IQR (depending on the selected method) for each column will be calculated 
+      independently. If any of the values in the selected columns are an outlier, the row will 
+      be flagged as an outlier (or dropped if drop = True).
+  drop:
+    type: boolean
+    description: Whether or not to drop the rows which are determined outliers. If false, a new
+      column will be created flagging outliers with a boolean.
+  method:
+    type: value
+    description: The method used to calculate outliers. Supported methods are
+      "iqr" which calculates the inter quartile range between the 1st and third quartile
+      (IQR) and flags values that are more than 1.5 * IQR from the median, 
+      "z-score" which calculates the z-score and flags values with a Z-score more than 
+      the provided threshold, and "threshold" which requires and manually set 
+      minimum and maximum threshold which are used to flag outliers.
+    
+      Default Value = "z-score"
+  min_threshold:
+    type: value
+    description: The minimum threshold for values that won't be flagged as outliers.
+      Required if method is "threshold".
+    is_optional: true
+  max_threshold:
+    type: value
+    description: The maximum threshold for values that won't be flagged as outliers.
+      Required if method is "threshold".
+    is_optional: true
+  max_zscore:
+    type: value
+    description: The maximum Z-score for values which will not be flagged as an outlier. 
+      Default Value = 2
+    is_optional: true
+example_code: |
```

Comment:
@kimballh  i think the example_code needs to be updated to reflect the new arguments you added

Reply:
@cpdough Done, I added an example for all three methods.